components/edit-collective/sections/Tiers.js

```diff
@@ -1,259 +1,246 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
Why changing this file?

Reply:
const [setSettings, { loading }] = useMutation(updateSettingsMutation); 

If I am not wrong , we can't use that in a class based approach  and hence I switch to functional based

Reply:
You're absolutely right, but let's move that to another PR. This is out of scope for this one

Reply:
Actually it has already been fixed on master

Reply:
I thought the component was already using `useState` in a class. As mentioned on Slack let's rather use `graphql` or the `Mutation` component for this PR, moving the component to hooks is out of scope and not especially something we want to do.